content/docs/command-reference/run.md

```diff
@@ -17,7 +17,7 @@ usage: dvc run [-h] [-q | -v] [-d <path>] [-n <name>] [-o <path>]
                command
 
 positional arguments:
-  command               Command to execute.
+  command               Command for the stage. Executed by default
```

Comment:
Help desc. update!

Reply:
I can submit a PR on the core repo if you agree, Ruslan. Cc @pared

Reply:
Looks really strange w/o a dot at the end

Reply:
My bad, I think this is actually a full sentence (with implicit subject & verb "It's").

Reply:
Fixing in #1540

Reply:
@jorgeorpinel you want to do the PR on core, or would you rather like us to do it?

Reply:
@jorgeorpinel, I think this is quite opposite. Rather, `--no-exec` is a special case.

Reply:
And, technically, the command may or may not execute (think of run-cache and already-run stages).

Reply:
> technically, the command may or may not execute (think of run-cache

If it's in the tun cache it means the results are re-applied at `dvc run` again. From the user's perspective this is the same as executing it again (instantly)

> you want to do the PR on core

Yes, will do... ⏳

Reply:
> Yes, will do... ⏳

Actually, since there's some controversy here, I'll let you guys decide @pared and @skshetry and if you do not change it just lmk in order to roll this one back.

Reply:
@jorgeorpinel, I don't see any difference between previous and this version. It only adds confusion (what does `by default` mean?).  It'd be better to explain the cases where it won't be executed.

Reply:
I think @skshetry is right, I don't think we need to explain when it's not executed, thats is what  `--no-exec` reference is for. Maybe we should replace `Only create stage file without actually running it.` -> `Create stage file without actually executing its command.` Since we are operating on `command` and `execute` in main ref

Reply:
@skshetry you don't see a difference between "Command to execute" and "Command for the stage" ? The former reflected run being a command to run commands (what it used to be), the latter reflects it being a helper to define stages (what it is now, see the cmd ref Description).

I thought we were just discussing the "Executed by default" which I don't have a strong opinion on, tbh. Removed in #1550 (let's continue this part of the discussion there?)

> Maybe we should replace Only create stage file without actually running it. -> Create stage file without actually executing its command.

@pared I agree. Or use the text from the cmd ref: `create a stage file, but do not execute the `command` defined in
  it, nor cache dependencies or outputs`